resources/home/dnanexus/eggd_artemis.py

```
@@ -0,0 +1,438 @@
+                expiry_date,
+                job_output,
+            ): sample
+            for sample, sample_data in file_data.items()
+        }
+
+        for future in concurrent.futures.as_completed(concurrent_jobs):
+            # access returned output as each is returned in any order
+            try:
+                data = future.result()
+                all_sample_outputs[data["sample"]] = data
+            except Exception as exc:
```

Ruff (0.8.0)

Do not catch blind exception: `Exception`

(BLE001)